lib/features/login/data/network/config/authorization_interceptors.dart

```diff
@@ -94,20 +91,20 @@ class AuthorizationInterceptors extends QueuedInterceptorsWrapper {
         _token!.refreshToken
       );
 
-      final accountCurrent = await _accountCacheManager.getSelectedAccount();
+      final currentAccount = await _accountCacheManager.getCurrentAccount();
 
-      await _accountCacheManager.deleteSelectedAccount(_token!.tokenIdHash);
```

Comment:
As I understand, we failed to delete account?

Reply:
It doesn't mean it can't be erased. But we mistakenly delete it when deleting according to the token variable stored in memory